.github/dependabot.yml

```diff
@@ -0,0 +1,14 @@
+version: 2
+updates:
+  - package-ecosystem: pip
+    directories:
+      - /utils/cq/requirements.txt
+    schedule:
+      interval: daily
+    groups:
+      python-packages:
+        patterns:
+          - pylint
+          - yamllint
+          - isort
+          - codespell
```

Comment:
Curious why you only included these packages in the group and not say all of the packages in the entire file.

Reply:
(or forget to update this when we pin a new package version)